REVIEW_GUIDELINES.md

```diff
@@ -33,7 +33,28 @@ Stable charts should not depend on charts in incubator.
 
 ## Names and Labels
 
-Resources and labels should follow some conventions. The standard resource metadata should be this:
+Resources and labels should follow some conventions.
+
+**NOTE:** 
+*The guidelines for standard labels have changed, but due to labels being immutable, charts with the
+legacy labels should continue to use the old label-style, while new charts should start using the updated labels.*
```

Comment:
In fact, for charts still using extensions/v&beta1 or apps/v1beta1, it is not immutable so there is a opportunity for them to upgrade.

Reply:
I agree with comments like https://github.com/helm/charts/pull/9332#pullrequestreview-176905476. The consensus in meetings so far has been that we shouldn't unnecessarily make breaking changes to charts, and updating to the new labels isn't a good enough reason. But if there are going to be significant enough breaking changes to a chart anyway, it could be a good time to update to the new labels in the same PR.